packages/blade/src/components/ActionList/docs/stories.ts

```diff
@@ -83,15 +83,23 @@ const ActionListItem = `
     ActionListItem, 
     ActionListItemIcon, 
     ActionListItemText, 
+    ActionListItemBadgeGroup,
+    ActionListItemBadge,
     HomeIcon 
   } from '@razorpay/blade/components';
 
   function App(): React.ReactElement {
     return (
-    <Box>
+    <Box backgroundColor="surface.background.level2.lowContrast">
       <ActionList>
         <ActionListItem 
           title="Title"
+          titleSuffix={(
+            <ActionListItemBadgeGroup>
+              <ActionListItemBadge>as: Keyword</ActionListItemBadge>
```

Comment:
can we have better stories like "With Badges for items" something on those lines? we can have all the components of ActionList with their props table listed and then different examples which are use cases centric but full examples instead of just individual sub-component examples? so that way it'll be easier for our consumers to scan

Reply:
I updated our AutoComplete controlled filtering example to use badges since it is mostly used in controlled filtering context to search on keywords

<img width="590" alt="image" src="https://github.com/razorpay/blade/assets/30949385/5713f532-5ec8-4324-850c-c3a6ffca9715">

Regarding ActionList components with tables, it already exists - https://61c19ee8d3d282003ac1d81c-lqbhyaeney.chromatic.com/?path=/docs/components-dropdown-actionlist--page

I added link to that AutoComplete filtering example with badges in ActionListItemBadge section